components/admin_console/admin_definition.test.jsx

```diff
@@ -18,7 +18,9 @@ const baseShape = {
 const fieldShape = {
     ...baseShape,
     key: yup.string().required(),
-    help_text: yup.string(),
+
+    // help_text: yup.string(), // Commented out since this doesn't work when help_text is a ReactNode
```

Comment:
Should we just remove this?

Reply:
I think I left it for documentation purposes. I'm 0/5 since this test will become unnecessary as soon as admin_definitions.jsx is migrated to TS anyway